aikau/src/main/resources/alfresco/pickers/SitePicker.js

```diff
@@ -0,0 +1,245 @@
+       * displayed in the picker.
+       *
+       * @instance
+       * @type {object}
+       */
+      widgetsForRootPicker: [
+         {
+            name: "alfresco/menus/AlfVerticalMenuBar",
+            config: {
+               widgets: [
+                  {
+                     name: "alfresco/menus/AlfMenuBarItem",
+                     config: {
+                        label: "picker.recentSites.label",
+                        publishTopic: "ALF_ADD_PICKER",
+                        publishPayload: {
+                           currentPickerDepth: 0,
+                           pickerLabel: "RecentSites",
+                           picker: [
+                              {
+                                 name: "alfresco/lists/AlfSortablePaginatedList",
+                                 config: {
+                                    itemsProperty: "",
+                                    loadDataPublishTopic: "ALF_GET_RECENT_SITES",
+                                    waitForPageWidgets: false,
+                                    widgets: [sitesView]
+                                 }
+                              }
+                           ]
+                        }
+                     }
+                  },
+                  {
+                     name: "alfresco/menus/AlfMenuBarItem",
+                     config: {
+                        label: "picker.favouriteSites.label",
+                        publishTopic: "ALF_ADD_PICKER",
+                        publishPayload: {
+                           currentPickerDepth: 0,
+                           pickerLabel: "FavouriteSites",
+                           picker: [
+                              {
+                                 name: "alfresco/lists/AlfSortablePaginatedList",
+                                 config: {
+                                    itemsProperty: "",
+                                    loadDataPublishTopic: "ALF_GET_FAVOURITE_SITES",
+                                    waitForPageWidgets: false,
+                                    widgets: [sitesView]
+                                 }
+                              }
+                           ]
+                        }
+                     }
+                  },
+                  {
+                     name: "alfresco/menus/AlfMenuBarItem",
+                     config: {
+                        label: "picker.allSites.label",
+                        publishTopic: "ALF_ADD_PICKER",
+                        publishPayload: {
+                           currentPickerDepth: 0,
+                           pickerLabel: "AllSites",
+                           picker: [
+                              {
+                                 name: "alfresco/lists/AlfSortablePaginatedList",
+                                 config: {
+                                    itemsProperty: "",
```

Comment:
Although the view will have a "noItemsMessage" we might also want to look at the other messages that are displayed by a list as they're document centric by default.